src/UserInterfaces/WindowsForms/Controls/TextView.cs

```diff
@@ -540,7 +540,7 @@ internal void SaveSelectionToStream(Stream stream)
             var modelPos = model.CurrentPosition;
             try
             {
-                var writer = new StreamWriter(stream, Encoding.Unicode);
+                var writer = new StreamWriter(stream, new UnicodeEncoding(false, false));
```

Comment:
Thanks for fixing this. I've been wondering where those BOMs were coming from, but never dedicated time to chase it down.